activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -7,7 +7,8 @@ module Reporting
       attr_accessor :gem_name
 
       # Outputs a deprecation warning to the output configured by
-      # <tt>ActiveSupport::Deprecation.behavior</tt>.
+      # <tt>ActiveSupport::Deprecation.behavior</tt>. By default
+      # +callstack+ is equal to caller(2).
```

Comment:
i think we need to indicate that `callstack` take `caller` method, what do you think?